ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/util/CommandParser.java

```diff
@@ -52,6 +54,8 @@
       Pattern.compile("\\s*SET\\s+'((?:[^']*|(?:''))*)'\\s*=\\s*'((?:[^']*|(?:''))*)'\\s*;\\s*");
   private static final Pattern UNSET_PROPERTY =
       Pattern.compile("\\s*UNSET\\s+'((?:[^']*|(?:''))*)'\\s*;\\s*");
+  private static final Pattern DROP_CONNECTOR =
+      Pattern.compile("\\s*DROP\\s+CONNECTOR\\s+((?:[^']*|(?:''))*)\\s*;\\s*");
```

Comment:
Why is the connector name portion of this regex escaping single quotes? I wouldn't expect those to be valid within an identifier.